diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -50,6 +54,9 @@ open class DiktatJavaExecTaskBase @Inject constructor(
         } else {
             main = "com.pinterest.ktlint.Main"
         }
+        if (diktatExtension.reporterType == "html") {
```

Comment:
Is it needed? If yes, why onlu for html?

Reply:
Others are already included in ktlint (plain, json, checkstyle)

Reply:
Oh, ok, that's strange. Leave a comment in code to explain this, pls.